bears/csv/CSVLintBear.py

```diff
@@ -17,6 +39,7 @@ class CSVLintBear:
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Syntax'}
+    ASCIINEMA_URL = 'https://asciinema.org/a/dlzd1y6vnwx7kk1a17p2gk7ll'
```

Comment:
When Its an error regarding row, I don't want to capture the remaining text
But when it is not, I want to capture all the text. How do I do this?